libs/ui/src/remove_card_page.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Main } from './main';
+import { Prose } from './prose';
+import { fontSizeTheme } from './themes';
+
+const RemoveCardImage = styled.img`
+  margin-right: auto;
+  margin-bottom: 5px;
```

Comment:
Design nit: I've always felt that the lock icon on the lock screen is a bit too close to the "[Machine Name] is Locked" text. I see you're just copying [that styling](https://github.com/votingworks/vxsuite/blob/ff31293e85377fb79e5cc760b40b522bc31e9da6/frontends/election-manager/src/screens/machine_locked_screen.tsx#L14) here, but what do you think of increasing that spacing a bit, both here and elsewhere?

Reply:
I played with it a bit when I made the component but didn't notice a difference / care too much, so sided with consistency. I would be pro- changing it in both places, because I also felt like it was slightly cramped.

Reply:
Let's save the change the lock screen for when we refactor those screens generally, and I'll increase the space on the remove card screen now.

Reply:
Sounds great!